src/components/GameController.tsx

```diff
@@ -55,100 +55,54 @@ export default class GameController extends PureComponent<Props> {
         return <GameProposedPage message="Waiting for opponent to accept game" />;
 
-      case playerA.WaitForPreFundSetupB:
-        return <GameProposedPage message="Waiting for opponent to accept game" />;
-
-      case playerA.ReadyToFund:
-        return <WaitingStep message="Ready to send funds" />;
-
-      case playerA.WaitForFunding:
+      case playerA.WAIT_FOR_FUNDING:
         return <WalletController />;
 
-      case playerA.ReadyToSendPostFundSetupA:
-        return <FundingConfirmedPage message="Sending acknowledgement to opponent" />;
-
-      case playerA.WaitForPostFundSetupB:
+      case playerA.WAIT_FOR_POST_FUND_SETUP:
         return <FundingConfirmedPage message="Waiting for opponent to acknowledge" />;
 
-      case playerA.ReadyToChooseAPlay:
+      case playerA.CHOOSE_PLAY:
         return <SelectPlayPage choosePlay={choosePlay} />;
 
-      case playerA.ReadyToSendPropose:
-        const state2 = applicationState as playerA.ReadyToSendPropose;
-        return <PlaySelectedPage message="Ready to send round proposal" yourPlay={state2.aPlay} />;
-
-      case playerA.WaitForAccept:
-        const state3 = applicationState as playerA.WaitForAccept;
+      case playerA.WAIT_FOR_ACCEPT:
         return (
-          <PlaySelectedPage message="Waiting for opponent to accept" yourPlay={state3.aPlay} />
-        );
-
-      case playerA.ReadyToSendReveal:
-        const state0 = applicationState as playerA.ReadyToSendReveal;
-        return (
-          <ResultPage
-            message="Waiting for resting"
-            yourPlay={state0.aPlay}
-            theirPlay={state0.bPlay}
-            result={state0.result}
-          />
+          <PlaySelectedPage message="Waiting for opponent to accept" yourPlay={applicationState.aPlay} />
         );
 
-      case playerA.WaitForResting:
-        const state1 = applicationState as playerA.WaitForResting;
+      case playerA.WAIT_FOR_RESTING:
         return (
           <ResultPage
             message="Waiting for resting"
-            yourPlay={state1.aPlay}
-            theirPlay={state1.bPlay}
-            result={state1.result}
+            yourPlay={applicationState.aPlay}
+            theirPlay={applicationState.bPlay}
+            result={applicationState.result}
           />
         );
 
-      case playerA.InsufficientFundsA:
-        return <WaitingStep message="About to conclude the game – you've run out of funds!" />;
-
-      case playerA.InsufficientFundsB:
-        return (
-          <WaitingStep message="About to conclude the game – your opponent has run out of funds!" />
-        );
-
-      case playerB.ReadyToSendPreFundSetupB:
-        return <WaitingStep message="Ready to send prefund setup" />;
+      case playerA.INSUFFICIENT_FUNDS:
+        // todo: add into the logic about who it was that ran out of funds
+        return <WaitingStep message="About to conclude the game – either you or your opponent has run out of funds!" />;
```

Comment:
Why does `Insufficient Funds` correspond to `WaitingStep`? Is there actually something we are waiting for?

Reply:
No - we just don't have a component for that yet, so are using `WaitingStep` temporarily.

Reply:
Great - just making sure :)